apache-rat-core/src/main/java/org/apache/rat/OptionCollection.java

```diff
@@ -209,23 +209,22 @@ private OptionCollection() {
             .converter(s -> ReportConfiguration.Processing.valueOf(s.toUpperCase()))
             .build();
 
-
-    /** TODO rework when commons-cli 1.7.1 or higher is available. */
-    static final DeprecatedAttributes DIR_ATTRIBUTES = DeprecatedAttributes.builder().setForRemoval(true).setSince("0.17")
-            .setDescription("Use '--'").get();
     /**
      * Ths option to signal the end of an argument list and the start of the directory/archive arguments.
      */
+    // TODO rework when commons-cli 1.7.1 or higher is available.
```

Comment:
What is the stuff that needs reworking with the new CLI version? I thought part2 contains the changes that come in with the new commonscliversion? Could you add this context to the PR? Thx